dffml/util/net.py

```diff
@@ -111,7 +128,9 @@ def cached_download(
     target_path = pathlib.Path(target_path)
 
     def validate_hash(error: bool = True):
-        filehash = hashlib.sha384(target_path.read_bytes()).hexdigest()
+        filehash = secure_hash(
+            target_path.read_bytes(), algorithm="sha384", encoding="no"
```

Comment:
This is a place where if we went with the `instance()` approach in `secure_hash()`, we wouldn't need to explictly set the encoding keyword argument here

Reply:
Encoding argument removed, isinstance is added.